build/build-indices.ts

```diff
@@ -21,7 +21,7 @@ const langs = {
   const index = client.initIndex(indexName)
   index.clearObjects().then(() => {
     const files = fg.sync(`website/docs/${lang}/*.md`)
-    let indices = []
+    const indices: Record<string,any>[] = []
```

Comment:
Space after ','

Reply:
Okay, I will add a lint rules later.

Reply:
Also for `Array` types use `Array<T>` instead of `Record`